### DISCUSSION TITLE
Poll: removal of morale loss of same food

### DISCUSSION THREAD

**Discussion by VissValdyr** [cataclysmbn/Cataclysm-BN#4589] [Polls]
For some we inherited the function of food being less and less boosty for morale if you eat the same all the time.
But since we have a lot of recipes that uses a wide array of ingredients (wood soup), we can't really say, that this is the same food all the time. And there are a lot of people who like eating the same food all the time.
Also since we raised the morale bonus to 12 hours, it is more than ever important to gameplay.
Food will always give 3x the morale boost. So drinking 3 beer and 3 whiskey would give morale boost for both. Drinking 4x beer would give only like drinking it 3 times.

**Comment by VissValdyr** [comment]
@scarf005 and @chaosvolt . I already promoted the poll on reddit. Can anyone of you post it on discord? I want to reach as many people as possible

**Comment by Firestorm01X2** [comment]
I think idea itself is fine. Also it gives reason to try new rcpepies and hunt for new ingidients.
If system is not fine then maybe in that case it better to just balance that mehcanics instead.

**Comment by VissValdyr** [comment]
I wasn't sure what people would prefer, that's why I made the poll. I am content with both answers

**Comment by chaosvolt** [comment]
I'm not sure I care either way to be honest. How long does the food morale reduction currently last? Telling the player how permanent an effect it actually is would be good here, so players can actually be informed about whether they want it completely removed or just set to wear off faster (which you didn't give as an option anyway).

**Reply by VissValdyr** [reply]
To be honest, I don't know the timer. It is less than 24 hours.

**Reply by chaosvolt** [reply]
Then that talks me into a no vote since that's already plenty fast enough of a recovery that it'd be about what I'd set it to if it felt too high. Well, technically yes but you know what I mean, weh.

**Comment by VissValdyr** [comment]
The votes are quite clear. So let's leave it as is

**Comment by Fl414r** [comment]
Why just not make this like balance mod or one of "debug" settings like crafting range?
